samples/rulesExtensions.sarif

```diff
@@ -0,0 +1,188 @@
+                },
+                "defaultConfiguration": {},
+                "properties": {
+                  "tags": [
+                    "maintainability"
+                  ],
+                  "kind": "problem",
+                  "precision": "very-high",
+                  "id": "js/unused-local-variable-1",
+                  "problem.severity": "recommendation"
+                }
+              },
+              {
+                "id": "com.lgtm/python-queries:py/unnecessary-pass-1",
+                "name": "com.lgtm/python-queries:py/unnecessary-pass",
+                "shortDescription": {
+                  "text": "com.lgtm/python-queries:py/unnecessary-pass shortDescription 1.0.0"
+                },
+                "fullDescription": {
+                  "text": "com.lgtm/python-queries:py/unnecessary-pass fullDescription 1.0.0"
+                },
+                "help": {
+                  "text": "TextHelp"
+                },
+                "helpUri": "http://lgtm.com",
+                "defaultConfiguration": {}
+              }
+            ]
+          }
+        ]
+      },
+      "results": [
+        {
+          "ruleId": "js/unused-local-variable-1",
+          "rule": {
+            "guid": "00000000-0000-1111-9999-111111111111",
+            "toolComponent": {
+              "index": 0
+            }
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "file.js"
+                },
+                "region": {
+                  "startLine": 1
+                }
+              }
+            }
+          ],
+          "message": {
+            "text": "Unused variable foo."
+          }
+        },
+        {
+          "ruleIndex": 0,
+          "rule": {
+            "guid": "00000000-0000-1111-9999-111111111111",
+            "toolComponent": {
+              "index": 0
+            }
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "file.js"
+                },
+                "region": {
+                  "startLine": 2
+                }
+              }
+            }
+          ],
+          "message": {
+            "text": "Unused variable foo."
+          }
+        },
+        {
+          "rule": {
+            "id": "js/unused-local-variable-1",
+            "toolComponent": {
+              "guid": "e0000000-0000-1111-9999-000000000000"
+            }
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "file.js"
+                },
+                "region": {
+                  "startLine": 3
+                }
+              }
+            }
+          ],
+          "message": {
+            "text": "Unused variable foo."
+          }
+        },
+        {
```

Comment:
Lookup tool component via `guid` and rule via `index`. This does not work at the moment.